pkg/cmd/orders.go

```diff
@@ -46,9 +46,17 @@ var getOrderCmd = &cobra.Command{
 		}
 
 		orderID, err := cmd.Flags().GetString("order-id")
+		if err != nil {
+			return fmt.Errorf("can't get the order-id from flags: %w", err)
+		}
+
+		symbol, err := cmd.Flags().GetString("symbol")
 		if err != nil {
 			return fmt.Errorf("can't get the symbol from flags: %w", err)
 		}
+		if symbol == "" {
+			return fmt.Errorf("symbol is not found")
```

Comment:
use `symbol is required` ?

Reply:
yes, error returned from binance before fix is

```
FATAL cannot execute command error=<APIError> code=-1105, msg=Parameter 'symbol' was empty.
```

Reply:
yeah I mean let's use the error message "symbol is required"

Reply:
symbol is required param in viber, so that I remove that check, both in get-order and list-orders command